timeless/items/views.py

```diff
@@ -11,11 +12,17 @@
 BP = Blueprint("items", __name__, url_prefix="/items")
 
 
-@BP.route("/")
-def list_items():
-    """ List the items """
-    items = [{"id": 1}, {"id": 2}]
-    return render_template("items/list.html", items=items)
+class ItemListView(ListView):
+    """ List the Items """
+    model = None
+    template_name = "items/list.html"
+    items = None
+
+    def get_object_list(self):
+        return self.items
```

Comment:
@paulodamaso It will break logic how `ListView` works. So, you fix your test and break view logic.